sparse/impl/KokkosSparse_sptrsv_solve_impl.hpp

```diff
@@ -1290,7 +1290,7 @@ struct SptrsvWrap {
     using LowerTPFunc = FunctorTypeMacro(TriLvlSchedTP1SolverFunctor, true, BlockEnabled);
 
 #if defined(KOKKOSKERNELS_ENABLE_SUPERNODAL_SPTRSV)
-    using namespace KokkosSparse::Experimental;
+    using KokkosSparse::SPTRSVAlgorithm;
```

Comment:
```suggestion
```
Since you are calling the struct with a fully qualified namespace we can just remove this